utils/sendpart/sendpart.go

```diff
@@ -61,6 +61,20 @@ func POST(c *httputil.Client, data DataMultipartWriter, v interface{}, url strin
 	return json.DecodeStream(body, v)
 }
 
+// PATCH sends a PATCH request using client to the given URL and unmarshals the
+// body into v if it's not nil. It will only send using multipart if needed.
+// It is equivalent to calling Do with http.MethodPost
+func POST(c *httputil.Client, data DataMultipartWriter, v interface{}, url string) error {
+	return Do(http.MethodPost, c, data, v, url)
```

Comment:
The `http.Method*` constants probably shouldn't be used here to avoid importing `net/http` just for those constants.